src/azul/service/elasticsearch_service.py

```diff
@@ -504,61 +504,75 @@ def transform_summary(self,
                                          post_filter=False,
                                          entity_type=entity_type)
 
-        # Add a total file size aggregate
-        es_search.aggs.metric(
-            'total_size',
-            'sum',
-            field='contents.files.size_')
-
-        # Add a cell count aggregate per organ
-        es_search.aggs.bucket(
-            'group_by_organ',
-            'terms',
-            field='contents.cell_suspensions.organ.keyword',
-            size=config.terms_aggregation_size
-        ).bucket(
-            'cell_count',
-            'sum',
-            field='contents.cell_suspensions.total_estimated_cells_'
-        )
-
-        # Add a total cell count aggregate
-        es_search.aggs.metric(
-            'total_cell_count',
-            'sum',
-            field='contents.cell_suspensions.total_estimated_cells_'
-        )
-
-        # Add an organ aggregate to the Elasticsearch request
-        es_search.aggs.bucket(
-            'organTypes',
-            'terms',
-            field='contents.samples.effective_organ.keyword',
-            size=config.terms_aggregation_size
-        )
-
-        for cardinality, agg_name in (
-            ('contents.specimens.document_id', 'specimenCount'),
-            ('contents.donors.genus_species', 'speciesCount'),
-            ('contents.files.uuid', 'fileCount'),
-            ('contents.donors.document_id', 'donorCount'),
-            ('contents.projects.laboratory', 'labCount'),
-            ('contents.projects.document_id', 'projectCount')
-        ):
-            es_search.aggs.metric(
-                agg_name, 'cardinality',
-                field=cardinality + '.keyword',
-                precision_threshold="40000")
+        cardinality_precision_threshold = 40000
```

Comment:
you might get a future review comment about moving this closer to where its used, ~ln 552